homeassistant/components/transmission/__init__.py

```diff
@@ -99,7 +100,7 @@
     return True
 
 
-def get_api(host, port, username=None, password=None):
+async def get_api(host, port, username=None, password=None):
     """Get Transmission client."""
     try:
         api = transmissionrpc.Client(host, port=port, user=username, password=password)
```

Comment:
This need to be scheduled on the executor thread pool.

Reply:
I need to pass hass here to be able to do that correct?

Reply:
Yes.